package.json

```diff
@@ -17,7 +17,7 @@
     "@types/node": "10.17.27",
     "@types/prettier": "2.6.0",
     "@types/semver": "^7.5.6",
-    "aws-cdk": "2.45.0",
+    "aws-cdk": "2.139.0",
```

Comment:
Was unable to deploy using local cdk due to version mismatch between this and aws-cdk-lib in dependencies. So matched to line 29